src/index.js

```diff
@@ -0,0 +1,150 @@
+    state.city = updatedCity;
+    currentCity.textContent = state.city;
+};
+
+const updateresetCity = () => {
+    const updatedCity = document.getElementById('cityInput');
+    const currentCity = document.getElementById('currentCity');
+    state.city = "Seattle";
+    currentCity.textContent = state.city;
+    updatedCity.value = state.city;
+};
+
+const updateSky = () => {
+    const sky = document.getElementById('sky');
+    const skyValue = document.getElementById('skies').value;
+    if (skyValue === 'sunny') {
+        sky.textContent = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️'
+    } else if (skyValue === 'cloudy') {
+        sky.textContent = '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️'
+    } else if (skyValue === 'rainy') {
+        sky.textContent = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧'
+    } else if (skyValue === 'snowy') {
+        sky.textContent = '❄️☃️❄️☃️❄️☃️❄️☃️❄️☃️❄️☃️❄️'
+    }
+};
+
+
+const updatelandScape = () => {
+    const sky = document.getElementById('sky');
+    if (state.temp > 80) {
+        tempInfo.style.color = 'red'
+        sky.textContent = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️'
+        landScape.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂'
+    } else if (state.temp > 70) {
+        tempInfo.style.color = 'orange'
+        sky.textContent = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️'
+        landScape.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷'
+    } else if (state.temp > 60) {
+        tempInfo.style.color = 'yellow'
+        sky.textContent = '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️'
+        landScape.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃'
+    } else if (state.temp > 50) {
+        tempInfo.style.color = 'green'
+        sky.textContent = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧'
+        landScape.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲'
+    } else if(state.temp < 49) {
+        tempInfo.style.color = 'teal'
+        landScape.textContent = '❄️☃️❄️☃️❄️☃️❄️☃️❄️☃️❄️☃️❄️'
+    }
+};
+
+const findLatitudeAndLongitude = () => {
+    let latitude, longitude;
+    axios
+    .get('https://weather-report-proxy-server.herokuapp.com/location', {
+        params: {
+        q: state.city,
+        },
+    })
+    .then((response) => {
+        latitude = response.data[0].lat;
+        longitude = response.data[0].lon;
+
+        // make the next API call here!
+        findTemperature(latitude, longitude);
+    })
+    .catch((error) => {
+        console.log('error in findLatitudeAndLongitude!');
+    });
+};
+
+const findTemperature = (latitude, longitude) => {
+    axios
+    .get('https://weather-report-proxy-server.herokuapp.com/weather', {
+        params: {
+        lat: latitude,
+        lon: longitude,
+        },
+    })
+    .then((response) => {
+        const kelvin = response.data['current']['temp'];
+        state.temp = convertKelvinToFahrenheit(kelvin);
+        updateWeather();
+    })
+    .catch((error) => {
+        console.log('error in weather info!');
+    });
+};
+
+const convertKelvinToFahrenheit = (kelvin) => {
+    let fahrenheit;
+    fahrenheit = parseInt(1.8 * (kelvin - 273) + 32);
+    return fahrenheit;
```

Comment:
Your method name is descriptive and I like that you put this logic in a helper method even though it's not long. It helps make your code more readable where you call `convertKelvinToFahrenheit`.

Here, to make this method more concise, I would just directly return the conversion like this:
```javascript
const convertKelvinToFahrenheit = (kelvin) => {
    return parseInt(1.8 * (kelvin - 273) + 32);
```